docs/sources/configuration/_index.md

```diff
@@ -667,16 +667,14 @@ alertmanager_client:
   [basic_auth_username: <string>]
   [basic_auth_password: <secret>]
 
-  # Optional `Authorization` header configuration.
-  authorization:
-    # Sets the authentication type.
-    [type: <string> | default: Bearer]
-    # Sets the credentials. It is mutually exclusive with
-    # `credentials_file`.
-    [credentials: <secret>]
-    # Sets the credentials to the credentials read from the configured file.
-    # It is mutually exclusive with `credentials`.
-    [credentials_file: <filename>]
+  # Sets the authentication type.
```

Comment:
The `basic_auth_username` and `basic_auth_password` mentioned just above should be on the same level as the ones you changed here I think. There is also a TLS field on NotifierConfig whose fields are not added here.

Reply:
The ```basic_auth_username``` and ```basic_auth_password``` are on the same level. I just removed the ```authorization``` keyword and moved the config options to the left.

Yes, the TLS options are missing. I could add those too